style.css

```diff
@@ -0,0 +1,53 @@
+body{
+    text-align: center;
+    background-color: grey;
+}
+
+body > h1 >p{
+    color: white;
+    
+}
+
+.form-page{
+    
+    width: 600px;
+    padding: 20px;
+    margin: 50px 30%;
+    background-color: white;
+    text-align: left;
+}
+
+
+
```

Comment:
please, i would suggest you Add a professional Readme.md for your project and well documented commit message